src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  --lite-blue: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
```

Comment:
Fix it
<img width="380" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/7a780feb-b5cb-4428-9174-08fc9ccb1a66">
```suggestion
```